syntax = "proto3";

option go_package = "google.golang.org/grpc/examples/calculation-ingolang/src/protos/calculationpb";

package calculationpb;

service CalculationService {

    //Unary API
    rpc AddNumbers (AddRequest) returns (AddResponse) {};
}

message AddRequest {
    int32 value1 = 1;

    int32 value2 = 2;
}

message AddResponse {
    int32 sum = 1;
}
